src/components/PermissionsManagement/UserRoleTab.jsx

```diff
@@ -25,6 +25,7 @@ export const permissionLabel = {
   addTask: 'Add Task',
   deleteTask: 'Delete Task',
   editTask: 'Edit Task',
+  suggestTask: 'Suggest changes on a task',
```

Comment:
For consistency, please change it to `Suggest Changes on a Task`

Reply:
Done